introduction.md

```diff
@@ -1,2 +1,8 @@
 I am Max, that all :P
-My favorite Code School pathes are Ruby and JavaScript
\ No newline at end of file
+My favorite Code School pathes are Ruby and JavaScript
+
+Deadly skills
+* GO
+* Ruby
+* JavaScript
+* Git
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?